frontend/components/CardBadge/index.jsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styles from './styles';
+
+/**
+ * @param {Array} badgeInfo Array of badge image urls
+ * @returns {JSX}
+ */
+const CardBadge = ({ badgeInfo }) => {
+  const images = badgeInfo.map((image, index) => (
+    <img className={styles.image} src={image} alt="" key={index.toString()} />
```

Comment:
why is it better to use the generic img tag rather than the Image component from the Shopgate common/components library?

Reply:
It isn't. Our Image component supports lazy and progressive loading of images. Updated to use Image component.